src/main/java/by/exadel/internship/controller/InternshipController.java

```diff
@@ -34,4 +32,29 @@ public GuestInternshipDTO getInternshipById(@PathVariable UUID internshipId) {
         return internshipService.getById(internshipId);
 
     }
+
+    // I think we need to come up with another mapping
+    @GetMapping("/deletedInternships")
+    @ApiOperation("Return List of deleted Internships")
+    public List<GuestInternshipDTO> getDeletedInternshipList(){
+        return internshipService.getAllDeleted();
+    }
+
+    @GetMapping("/deletedInternships/{internshipId}")
+    @ApiOperation("Return deleted Internship by ID")
+    public GuestInternshipDTO getDeletedInternship(@PathVariable UUID internshipId){
+        return internshipService.getDeletedInternshipById(internshipId);
+    }
+
+    @DeleteMapping("/{internshipId}")
+    @ApiOperation("Delete Internship by ID")
+    public void deleteInternship(@PathVariable UUID internshipId){
+        internshipService.deleteInternshipById(internshipId);
+    }
+
+    @PutMapping("/returnInternship/{internshipId}")
+    @ApiOperation("Return deleted Internship to List of Internship")
```

Comment:
```suggestion
    @ApiOperation("Get removed internships")
```

Reply:
I meant a different meaning, but already renamed it for better readability